doc/cli.markdown

```diff
@@ -320,11 +320,31 @@ Examples:
 
 ### Options
 
-#### -v, --verbose
+#### --fields FIELDS
```

Comment:
This looks like a `cli.markdown` file that was generated for v13, not v12. It will probably fail the build with "uncommited changes"...

Reply:
Ooops

Reply:
Re-pushed now, sorry.